julia/docs/src/index.md

```diff
@@ -0,0 +1,97 @@
+# FlatBuffers.jl Documentation
+
+#### Overview
```

Comment:
It be good to have most of the docs under `docs/` that are user facing, so they end up as part of the website etc.

Reply:
This is basically because I've gone and added the whole Julia package in here. I already have an external link to these documents in the `docs/` folder, so you can get to them from the website. Until we are actually releasing the Julia package out of the `flatbuffers` project, it might be best to keep the documentation for that package along with it and link to it externally for now?